boltdesignsystem/bolt
# DS-208: remove "full" prop from bolt-banner

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-208](https://pegadigitalit.atlassian.net/browse/DS-208)

## Summary

Remove the prop "full" from bolt-banner.

## Details

There was an error in the Twig logic causing the "--full" class to always be added to the Twig rendered Banner component. This caused the Banner to briefly appear full viewport width until JS loads and the web component boots up.

Rather than fix the logic, we're just removing the "full" prop because @mikemai2awesome mentioned there's a bug with positioning things full viewport width and this prop is not currently in use anywhere downstream.

Also added a Banner demo to the [Academy Topic template](https://feature-ds-208-remove-banner-full.boltdesignsystem.com/pattern-lab/?p=pages-acd-topic-detail-page) to show the component in action.

## How to test

Check out feature branch and review Banner component in pattern lab. Perhaps check out a couple Pega properties. I searched gitlab as well as I could and did not notice it being used.

## Release notes

Deprecate "full" prop on Banner component.
